app/forms/idv/doc_pii_form.rb

```diff
@@ -45,6 +46,8 @@ def validate_pii
         errors.add(:pii, dob_error, type: :dob_error)
       elsif !dob_meets_min_age?
         errors.add(:pii, dob_min_age_error, type: :dob_min_age_error)
+      elsif address1.blank?
```

Comment:
Does pii always contain address1 key?

Reply:
I think that this is the form that gets the pii we read off the id and this is what we want to check with this ticket: Whether that data contains `address1`. I'm not 100% confident though.

Reply:
I am not sure neither, so asked, if it missing that key, we will get null pointer when call address1.blank?

Reply:
the initializer only fetch it from pii hash, that's why i asked whether it always contains the key.

for trueid :

```ruby
PII_INCLUDES.each do |true_id_key, idp_key|
  pii[idp_key] = true_id_product[:IDAUTH_FIELD_DATA][true_id_key]
end
```
a well formed response sure contains non null entry.

Reply:
Can you say more?

- I looked at `app/services/doc_auth/lexis_nexis/responses/true_id_response.rb` where that code excerpt is from and it looks like that excerpt translates key names like: `'Fields_AddressLine1'` to `:address1`
- I looked at `app/services/doc_auth/response.rb` as well, but didn't see anything relevant.
- I definitely don't fully understand how `pii_from_doc`, which is the source of that hash (?), is used, it's an important method though.